asset-transfer-basic/application-javascript/package.json

```diff
@@ -15,8 +15,8 @@
     "author": "Hyperledger",
     "license": "Apache-2.0",
     "dependencies": {
-        "fabric-ca-client": "^2.1.0",
-        "fabric-network": "^2.1.0"
+        "fabric-ca-client": "2.2.0",
```

Comment:
We usually keep the ^ prefix so that it will use the latest v2.x.x version, rather than v2.2.0 specifically.